packages/bits/src/styles/nui-framework-variables.less

```diff
@@ -14,7 +14,7 @@
 // == Fonts
 @font-family-open-sans: "Open Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;
 @font-family-source-code-pro: "Source Code Pro", monospace;
-@font-family-roboto: "Roboto", @font-family-open-sans;
+@font-family-roboto: "Roboto";
```

Comment:
It seems like we should keep the fallback font in case roboto doesn't get loaded for some reason.